### PR TITLE
chore(FEC-9189): add tests for HLS and DASH adapters following FEC-8975

### PR DESCRIPTION
**Pull request by RoyBregman**
### Description of the Changes

in ticket FEC-8975 functionallity was added to hls and dash to support kava but didn't have new unit tests to support it

solves FEC-9189

### CheckLists

- [X] changes have been done against master branch, and PR does not conflict
- [X] new unit / functional tests have been added (whenever applicable)
- [X] test are passing in local environment
- [ ] Travis tests are passing (or test results are not worse than on master branch :))
- [ ] Docs have been updated

### CODE REVIEW

File: test/src/dash-adapter.spec.js [hunk 1]
```diff
@@ -231,6 +254,97 @@ describe('DashAdapter: load', () => {
...
+    } catch (e) {
+      done(e);
+    }
+  });
+
+  it('should check targetBuffer in LIVE and restricted by bufferingGoal', done => {
+    try {
+      dashInstance = DashAdapter.createAdapter(video, liveSource, config);
+      video.addEventListener(EventType.PLAYING, () => {
+        dashInstance._shaka.getConfiguration().streaming.bufferingGoal = 10;
```
Comment (Yuvalke) [inline test/src/dash-adapter.spec.js:332]: Shaka has configure method as you used in the previous test

File: test/src/dash-adapter.spec.js [hunk 2]
```diff
@@ -231,6 +254,97 @@ describe('DashAdapter: load', () => {
...
+    } catch (e) {
+      done(e);
+    }
+  });
+
+  it('should check targetBuffer in LIVE', done => {
+    try {
+      dashInstance = DashAdapter.createAdapter(video, liveSource, config);
+      video.addEventListener(EventType.PLAYING, () => {
+        dashInstance._shaka.configure({streaming: {bufferingGoal: 120}});
```
Comment (Yuvalke) [inline test/src/dash-adapter.spec.js:310]: Is there a reason to configure on `playing`? we don't do that on the player so I think it'll be better to do that before.

File: test/src/dash-adapter.spec.js (hunk 1 above)
Comment (RoyBregman) [reply test/src/dash-adapter.spec.js:332]: done

File: test/src/dash-adapter.spec.js (hunk 2 above)
Comment (RoyBregman) [reply test/src/dash-adapter.spec.js:310]: done

**Pull request merged**